app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :set_user_and_post, only: %i[create]
+
+  load_and_authorize_resource
+
+  def index
+    @comments = Comment.where(post_id: params[:post_id], user_id: params[:user_id])
+  end
+
+  def create
+    @comment = @post.comments.build(comment_params.merge(user_id: @user.id))
+
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: @comment.errors, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
- [ ] Great job making the requested changes by the previous reviewer, how ever a new error has been introduced still blocking the creation of a comment for a user, kindly fix this. This error occurs when you try to assign a value to an attribute that does not exist in the model. 
The error message indicates that the Comment model does not have an attribute named author_id. 

Error 
![image](https://github.com/Chimoneg27/Rails-Blog-App/assets/81932478/bbe64121-bc16-49cf-9b2a-ed02b28f3479)

Screencast
[cinnamon-2023-12-16T154217+0200.webm](https://github.com/Chimoneg27/Rails-Blog-App/assets/81932478/8cd8f31d-182e-4c8e-813e-504617be547f)